helpers/compile/plugins/tscPlugin.ts

```diff
@@ -78,6 +78,13 @@ export const tscPlugin: (emitTypes?: boolean) => esbuild.Plugin = (emitTypes?: b
         await run(`tsc --project ${options.tsconfig} --paths null`)
       }
 
+      // for ecosystem tests, skip the bundling, just copy the types to go faster
```

Comment:
Slightly different from reality, ideally I'd like to have a quick mode for local dev like this and maybe a mode that is slower and does the full type bundling. They should be equivalent, but we never now.